### PR TITLE
Adds information on StyleGuideProvider prereq

### PR DESCRIPTION
**Pull request by mengtzu**
I'd half finished this when I saw the change to StyleGuideProvider in the slack channel.  Feel free to ignore this if you already have a doco revision in flight.  Otherwise happy to amend this one.

### CODE REVIEW

**Comment by markdalgleish** [comment]
This is looking great! To keep backwards compatibility, could you add `<StyleGuideProvider>` and keep `<SeekApp>` as an alias?

File: README.md [hunk 1]
```diff
@@ -229,9 +229,35 @@ import { HeartIcon, StarIcon } from 'seek-style-guide/react';
...
+- [Dropdown](http://seek-oss.github.io/seek-style-guide/dropdown)
+
+###StyleGuideProvider prerequisite
+
+You wrap your app in the `StyleGuideProvider` component to use any of the style guide components.  For example:
+
+```js
+render() {
+    return (
+      <StyleGuideProvider title={pageTitle} meta={meta.meta} link={meta.link} locale={locale}>
```
Comment (mjt01) [inline README.md]: For clarity I would make these prop values either strings or declare variables that demonstrate the interface. Thoughts?

File: README.md [hunk 2]
```diff
@@ -229,9 +229,35 @@ import { HeartIcon, StarIcon } from 'seek-style-guide/react';
...
+
+          ...
+
+        </div>
+      </StyleGuideProvider>
+    );
+}
+```
+
+`StyleGuideProvider`'s props are used to set the page head properties using Helmet.
```
Comment (mjt01) [inline README.md]: Can you make "Helmet" a link to the package please

File: README.md (hunk 1 above)
Comment (mengtzu) [inline README.md]: Done

File: README.md (hunk 2 above)
Comment (mengtzu) [inline README.md]: Done

**Comment by markdalgleish** [comment]
@mengtzu I'm happy to approve this once `StyleGuideProvider` is exported 🙂

**Comment by markdalgleish** [comment]
👍  Nice one!

**Pull request merged**